packages/core/content-releases/server/src/utils/index.ts

```diff
@@ -44,3 +54,27 @@ export const getEntryValidStatus = async (
     return false;
   }
 };
+
+/**
+ * Temporal function to get the entryId from documentId with locale
+ * This is needed because some services still need the entryId to work
+ */
+export const getEntryId = async (
+  {
+    contentTypeUid,
+    documentId,
+    locale,
+  }: { contentTypeUid: UID.ContentType; documentId?: string; locale?: string },
+  { strapi }: { strapi: Core.Strapi }
+) => {
+  if (documentId) {
+    const document = await strapi.documents(contentTypeUid).findOne({ documentId, locale });
+
+    return document?.id;
+  }
+
+  // documentId is not defined in singleTypes, so we want to get the first document
+  const document = await strapi.documents(contentTypeUid).findFirst({ locale });
+
+  return document?.id;
```

Comment:
Not really sure how this function is used but just curious if it is useful to return undefined or would it be more useful to throw an error?